src/cli/commands/run.js

```diff
@@ -135,7 +136,7 @@ export async function run(config: Config, reporter: Reporter, flags: Object, arg
           stage,
           config,
           cmd: cmdWithArgs,
-          cwd: flags.into || config.cwd,
+          cwd: realpathSync(flags.into || config.cwd),
```

Comment:
Can you instead use `path.resolve`? I don't think we want to use `realpath` here

Reply:
Pretty sure i tried it and it didnt work

Reply:
Doesn't work for me I get an error:
> error An unexpected error occurred: "realpathSync is not defined".

Same error when qualifying fs.realpathSync

`path.resolve` also didn't fix the issue, note the double node_modules below

> Cannot find module 'E:\projects\vscode-sqltools\node_modules\node_modules\vscode\bin\install'